app/components/App.tsx

```diff
@@ -179,22 +171,21 @@ class App extends Component {
           <div className="container">
             <Routes />
           </div>
-          <PopUpMessage />
```

Comment:
This failed typechecks, because it expects a prop, and is actually hidden unless the prop is passed, so I see no harm in removing it.

Reply:
Hmm, I took a look at the component definition, and I think your statement is incorrect. It looks like it uses a Redux `connect()`, so the props are actually injected by Redux.

Reply:
Good point, fixing.